dll/cpl/sysdm/lang/bg-BG.rc

```diff
@@ -325,6 +325,10 @@ END
 
 STRINGTABLE
 BEGIN
+    IDS_EDITVARCAPTION "Edit "
```

Comment:
For which dialog(s) are these new strings needed? Can you show a screenshot?

Reply:
![Before NewVariable](https://user-images.githubusercontent.com/49964946/215233818-5e54d4d3-0564-41ad-882a-aa134df3058c.jpg)
![AfterNewVariable](https://user-images.githubusercontent.com/49964946/215233824-b91e3a39-582e-4b99-8f55-8dacea9ded0b.jpg)